clients/admin-ui/src/features/locations/transformations.ts

```diff
@@ -0,0 +1,34 @@
+import { Continent, Location } from "~/types/api";
+
+export const groupByContinent = (
+  locations: Location[]
+): Record<Continent, Location[]> => {
+  const byContinent: Record<string, Location[]> = {};
+  const allContinents = new Set(locations.map((l) => l.continent).sort());
+  allContinents.forEach((continent) => {
+    byContinent[continent] = locations
+      .filter((l) => l.continent === continent)
+      .sort((a, b) => a.name.localeCompare(b.name));
+  });
+  return byContinent;
+};
+
+export const groupByBelongsTo = (
```

Comment:
not used yet, but I think will be by https://ethyca.atlassian.net/browse/PROD-1493